pulsar-client-cpp/lib/Commands.cc

```diff
@@ -258,7 +258,8 @@ SharedBuffer Commands::newSubscribe(const std::string& topic, const std::string&
                                     const std::map<std::string, std::string>& metadata,
                                     const SchemaInfo& schemaInfo,
                                     CommandSubscribe_InitialPosition subscriptionInitialPosition,
-                                    bool replicateSubscriptionState, KeySharedPolicy keySharedPolicy) {
+                                    bool replicateSubscriptionState, KeySharedPolicy keySharedPolicy,
+                                    int priority_level) {
```

Comment:
I suggeste keeping the code style consistent with the existing code. i.e. use low camel case instead of snake case for variables.

Reply:
Thanks for your suggestion, I have fixed it in the latest commit.